doc/source/users_guide/trouble-shooting/trouble-shooting.rst

```diff
@@ -6,44 +6,121 @@
 Trouble Shooting 
 *********************
 
-In this chapter we give some guidance on what to do when you encounter some of the most common problems. We can't cover all the problems that a user could potentially have, but we will try to help you recognize some of the most common situations. And we'll give you some suggestions on how to approach the problem to come up with a solution.
+In this chapter we give some guidance on what to do when you encounter some of the most common problems.
 
-In general you will run into one of three type of problems:
+In general you may run into one of four types of problems:
 
 1. *case-creation*
 #. *setup-time*
 #. *build-time*
 #. *run-time*
 
-See the `CIME Trouble Shooting Guide <http://esmci.github.io/cime/users_guide/troubleshooting.html>`_ for some help on the first three.
-
+Start with the `CIME Trouble Shooting Guide <http://esmci.github.io/cime/versions/master/html/users_guide/troubleshooting.html>`_ , especially if you encounter one of the first three types of problems. The CIME troubleshooting guide also provides some useful tips regarding run-time errors. If this doesn't identify and solve your problem, then try some of the suggestions below for run-time errors.
 
 General Advice on Debugging Run time Problems
 =============================================
 
-Here are some suggestions on how to track down a problem while running. In general if the problem still occurs for a simpler case, it will be easier to track down.
+The model has been run for thousands and thousands of simulation years in many different configurations, both fully-coupled and in land-only modes, without problems.  If you have modified the model in any way, by using either different input datasets or new or modified code, then obviously that is the first place to look if you encounter an error. However, the model is not completely infallible, as noted below.
```

Comment:
I would delete "obviously", since that can sound condescending.

Reply:
Done